queries/models/projeto_subsidio_sppo/schema.yml

```
@@ -49,7 +49,7 @@ models:
               to: ref('tecnologia_servico')
               field: servico
               to_condition: "inicio_vigencia <= date('{{ var('date_range_end') }}') and (fim_vigencia is null OR fim_vigencia >= date('{{ var('date_range_start') }}'))"
-              where: "data BETWEEN greatest(date('{date_range_start}'), date('{DATA_SUBSIDIO_V15A_INICIO}')) and date('{date_range_end}')"
+              where: "data BETWEEN greatest(date('{date_range_start}'), date('{DATA_SUBSIDIO_V15B_INICIO}')) and date('{date_range_end}')"

```

Aqui não seria julho?

```suggestion
              where: "data BETWEEN greatest(date('{date_range_start}'), date('{DATA_SUBSIDIO_V15A_INICIO}')) and date('{date_range_end}')"

```